realm/realm-annotations-processor/src/test/resources/io/realm/AllTypesRealmProxy.java

```diff
@@ -170,6 +171,9 @@ private void injectObjectContext() {
             if (!proxyState.getAcceptDefaultValue$realm()) {
                 return;
             }
+            final Row row = proxyState.getRow$realm();
+            row.getTable().setLong(columnInfo.columnLongIndex, row.getIndex(), value, true);
```

Comment:
Yes, but the `Table` is only used when the setters are called from its constructor.
And, `row.getTable()` is just returning `Row`'s private field (not involve JNI call).

So I would like to leave as it is.